src/components/executive-education-2u/components/CourseSummaryCard.jsx

```diff
@@ -49,15 +50,37 @@ const CourseSummaryCard = ({ courseMetadata, enrollmentCompleted }) => {
             <Col xs={12} lg={4}>
               <div className="course-details">
                 <Row className="align-items-center">
-                  <Col className="small font-weight-light text-gray-500 justify-content-start">{enrollmentCompleted ? 'Start date:' : 'Available start date:'}</Col>
+                  <Col className="small font-weight-light text-gray-500 justify-content-start">
+                    <FormattedMessage
+                      id="executive.education.external.course.enrollment.page.course.start.date.placeholder"
+                      defaultMessage="{dateText}:"
+                      description="showing start date or available start date based on enrollment sattus while enrolling executive education course"
+                      values={{
+                        dateText: enrollmentCompleted
+                          ? ('Start date') : ('Available start date'),
+                      }}
+                    />
```

Comment:
I think we need to convert this to multiple FormattedMessage or intl.formatMessage due to the reason discussed offline.